Gemfile.lock

```diff
@@ -203,18 +201,17 @@ GEM
       ruby2_keywords (~> 0.0.1)
     mustermann-grape (1.0.2)
       mustermann (>= 1.0.0)
-    net-imap (0.4.1)
+    net-imap (0.4.6)
       date
       net-protocol
     net-pop (0.1.2)
       net-protocol
-    net-protocol (0.2.1)
+    net-protocol (0.2.2)
       timeout
     net-smtp (0.4.0)
       net-protocol
-    nio4r (2.5.9)
-    nokogiri (1.15.4)
-      mini_portile2 (~> 2.8.2)
+    nio4r (2.6.1)
+    nokogiri (1.15.5-x86_64-darwin)
```

Comment:
Was this intentional?

Reply:
No this PR is now breaking after a merge with main. going to resolve.